demo/authenticationTransaction.php

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
This Demo should come in readme file i guess 
Please check with concern person to verify